distribution/ddf-common/src/main/resources/etc/artemis.xml

```diff
@@ -20,21 +23,58 @@
 
     <core xmlns="urn:activemq:core">
 
+        <persistence-enabled>true</persistence-enabled>
+
         <!-- this could be ASYNCIO or NIO -->
         <journal-type>ASYNCIO</journal-type>
-        <paging-directory>${ddf.home}/data/artemis/paging</paging-directory>
-        <bindings-directory>${ddf.home}/data/artemis/bindings</bindings-directory>
-        <journal-directory>${ddf.home}/data/artemis/journal</journal-directory>
-        <large-messages-directory>${ddf.home}/data/artemis/large-messages</large-messages-directory>
-        <journal-min-files>10</journal-min-files>
+        <paging-directory>${ddf.home:.}/data/artemis/paging</paging-directory>
```

Comment:
ill make them the same the artemis uses the right side of the : as a default value